File: codekata/absolute beginner/772.js
/*
Let "A"  be a string. Remove all the whitespaces and find it's length.

Positive TestCase :
    Input-Lorem Ipsum
    Output-10
*/

function fun(x){
  var a=''
  for(i in x){
    if (x[i]==' '){
      continue
    }
    else{
      var a=a+x[i]
    }
  }
  console.log(a.length)
}

var b="Lorsem Ipsum"
fun(b)

File: codekata/absolute beginner/772.py
'''Let "A"  be a string. Remove all the whitespaces and find it's length.

Input Description:
A string is provide as an input

Output Description:
Remove all the whitespaces and then print the length of the remaining string.

Sample Input :
Lorem Ipsum
Sample Output :
10'''

A = input()
a = 0
for i in range(len(A)):
  if ord(A[i]) == 32:
    a += 1
print(len(A)-a)

